Fabric.Terminology.API/Scripts/Install-Terminology.ps1

```diff
@@ -1,15 +1,34 @@
-﻿param(
+param(
     [ValidateNotNullorEmpty()]
     [ValidateScript({
             if (!(Test-Path $_)) {
-                Write-DosMessage -Level "Error" -Message "$_ does not exist. Please enter valid path." -ErrorAction Stop
+                Throw "$_ does not exist. Please enter valid path."
```

Comment:
Why `throw` here?  I have done the same thing in places but why not use `Write-DosMessage` as it was used before?

Reply:
because DosInstallUtilities has not been installed yet...

Reply:
That is a good enough reason...